components/KitButton.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Image, Text, TouchableOpacity } from 'react-native';
+import KitText from './KitText';
+/*
+props expected:
+  style: {
+    button:{
+
+    },
+    image: {
+
+    }
+  }
+  
+  onPress
+  image
+
+  buttonBackgroundColor
+  buttonTextColor
```

Comment:
Same for text color, are there multiple colors we want to allow?

Reply:
Will also include in the constants file